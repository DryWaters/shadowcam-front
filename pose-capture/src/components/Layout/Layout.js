import React from "react";
import styles from "./Layout.module.css";

const Layout = props => (
  <div className={styles.layoutContainer}>
    {props.children}
  </div>
);

export default Layout;
