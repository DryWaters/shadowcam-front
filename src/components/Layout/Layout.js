import React from "react";
import Header from "../Header/Header";
import styles from "./Layout.module.css";

const Layout = props => (
  <div className={styles.layoutContainer}>
    <Header />
    {props.children}
  </div>
);

export default Layout;
