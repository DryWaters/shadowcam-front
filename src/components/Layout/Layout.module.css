.layoutContainer {
    background-color: #27292e;
    color: white;
    min-height: 100vh;
    font-family: "Open Sans", sans-serif;
}