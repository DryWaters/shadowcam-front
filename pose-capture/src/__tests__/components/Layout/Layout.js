import React from "react";
import { shallow } from "enzyme";
import Layout from '../../../components/Layout/Layout'

describe("Layout render", () => {
  it("Should render Layout component without crashing", () => {
    shallow(<Layout />);
  });

  it("Should match Layout component snapshot", () => {
    const wrapper = shallow(<Layout />);
    expect(wrapper).toMatchSnapshot();
  });
});