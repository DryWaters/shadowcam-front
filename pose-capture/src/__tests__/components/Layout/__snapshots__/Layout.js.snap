// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout render Should match Layout component snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Layout />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Connect(Header) />,
        undefined,
      ],
      "className": "layoutContainer",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      undefined,
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Connect(Header) />,
          undefined,
        ],
        "className": "layoutContainer",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        undefined,
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
